conf/broker.conf

```diff
@@ -1395,7 +1395,7 @@ fileSystemURI=
 transactionCoordinatorEnabled=false
 transactionMetadataStoreProviderClassName=org.apache.pulsar.transaction.coordinator.impl.MLTransactionMetadataStoreProvider
 
-# Transaction buffer take snapshot transaction count
+# Transaction Buffer will take a snapshot after the number of transaction operations reaches this value.
```

Comment:
```suggestion
# Transaction buffer takes a snapshot after the number of transaction operations reaches this value.
```

1. Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

2. Why capitalize B for buffer?